src/index.ts

```diff
@@ -17,10 +17,6 @@ import { ApiResponseValidator } from './validation/ApiResponseValidator';
  * @public
  */
 export function provideCore(config: AnswersConfig): AnswersCore {
-  if (config.apiKey && config.token) {
```

Comment:
Can we do something like `if ('apiKey' in config && 'token' in config) {
    throw new Error('Both apiKey and token are present. Only one authentication method should be provided');
  }` so that we can keep this error message?